src/components/Calculator.js

```diff
@@ -1,55 +1,44 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from './logic/calculate';
 import ButtonCalc from './buttonCalc';
 
 // eslint-disable-next-line react/prefer-stateless-function
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
+function Calculator() {
```

Comment:
- [ ] Kindly write this Calculator component using es6 syntax, so this becomes -> 

```
const Calculator  = () => { 
    ... 
}
```

- [ ] It would also be nice if the App component is also refactored into -> 

```
const App = () => { 
    ... 
}
```